packages/grafana-test-utils/src/server/index.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpHandler } from 'msw';
+import { setupServer } from 'msw/node';
+
+import allHandlers from '../handlers/all-handlers';
+
+const server = setupServer(...allHandlers);
```

Comment:
idea: perhaps it would make sense in some scenarios to be able to opt-out of the default handlers? Or would the consumer need to manually call `resetHandlers()` after setting up the server in that case?

Reply:
Hmm I think perhaps the advice would be to call `resetHandlers` with the handlers that you want to "override"/reset with ([it supports that](https://mswjs.io/docs/api/setup-server/reset-handlers/))

Reply:
Yep, that's perfectly fine! 👍